generator/generator.yml

```diff
@@ -132,11 +132,34 @@ modules:
 # Tested on Arista DCS-7010T-48 switch
 #
   arista_sw:
-    walk: 
+    walk:
       - sysUpTime
       - interfaces
       - ifXTable
       - 1.3.6.1.2.1.25.3.3.1.2 # hrProcessorLoad
       - 1.3.6.1.2.1.25.2.3.1.6 # hrStorageUsed
       - 1.3.6.1.4.1.30065.3.1.1 # aristaSwFwdIp
 
+# DD-WRT
+#
+# The list of SNMP OIDs to care about for DD-WRT can be found here: https://www.dd-wrt.com/wiki/index.php/SNMP#Known_OID.C2.B4s_via_SNMP
+#
+# Tested on  DD-WRT v3.0-r31825 (04/06/17) with an ASUS RT-AC66U
+
+  ddwrt:
+    walk:
+      - sysUpTime
+      - interfaces
+      - ifXTable
+      - 1.3.6.1.2.1.25.2 # hrStorage
+      - 1.3.6.1.4.1.2021.4 # ucdavis mem
+      # ucdavis laTable (load avg)
+      - 1.3.6.1.4.1.2021.10.1.1 # laIndex
```

Comment:
<img width="310" alt="screen shot 2017-04-24 at 14 15 53" src="https://cloud.githubusercontent.com/assets/569574/25336611/95abe740-28f8-11e7-887f-d65a8754d085.png">

Are you sure you're looking at the right MIB: http://www.oidview.com/mibs/2021/UCD-SNMP-MIB.html?